### ISSUE TITLE
undefined method `captures' for nil:NilClass

### ISSUE THREAD

**Issue by naina-verma** [projectatomic/vagrant-service-manager#365]
undefined method `captures' for nil:NilClass ,while doing install-cli for openshift version 
`vagrant  service-manager install-cli openshift --cli-version 3.1`
OS details:
`Tested on Windows 10 , Fedora 23`

Provider:
VirtualBox and/or Libvirt

Output of vagrant -v:

`Vagrant 1.7.4`

Output of vagrant plugin list:

`$ vagrant.exe  plugin list
landrush (1.1.0.beta.3)
- Version Constraint: 1.1.0.beta.3
  vagrant-registration (1.2.3)
- Version Constraint: 1.2.3
  vagrant-service-manager (1.3.0.dev)
- Version Constraint: 1.3.0.dev
  vagrant-share (1.1.5, system)
  vagrant-sshfs (1.1.0.dev)
- Version Constraint: 1.1.0.dev`

Output of vagrant service-manager box version:

`Atomic Developer Bundle (ADB) 2.2.2`

Steps to reproduce the issue:
1. Download latest ADB and did vagrant up,plugins installed are latest
2. install-cli command to download oc binary vagrant service-manager install-cli openshift
3. Error while downloading openshift binary  with version (Works fine with docker)

It should show valid error message if the version is invalid

**Comment by brgnepal** [comment]
@naina-verma  This is related to how proper error messaging should be.
It has been fixed by the PR https://github.com/projectatomic/vagrant-service-manager/pull/363.

Now, error message is :

```
Something went wrong to find the correct download link.
We recommend you to use '--cli-version' for your desired client version.
If still facing issues, please report to https://github.com/projectatomic/vagrant-service-manager.
```

**Comment by naina-verma** [comment]
@budhrg ,yes it should be showing proper error message. But, as per the error shows up looks like something is broken.

**Comment by hferentschik** [comment]
Resolved via pull request #363
